ci-operator/jobs/openshift/multiarch/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
+  - deepsm007
+  - jaypoulz
+  - r4f4
+  - dptp
+reviewers:
+  - dptp
```

Comment:
I don't think DPTP wants to be an explicit approver or reviewer here. We do not own the jobs.